app_src/ui/screens/stats_screen.py

```
@@ -437,6 +451,14 @@ def _do_clear_config(self):
         ConfigManager.write(ConfigManager.DEFAULT_CONFIG)
         db = ImageDatabase()
         db.clear_all()
+        try:
+            from android_notify.internal.java_classes import autoclass
+            PythonActivity = autoclass("org.kivy.android.PythonActivity")
+            context = PythonActivity.mActivity.getApplicationContext()
+            prefs = context.getSharedPreferences("update_checker_prefs", 0)
+            prefs.edit().clear().apply()
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)